src/app/min/kmong/components/carousel.tsx

```diff
@@ -0,0 +1,257 @@
+import h3 from '@public/webp/8-3.webp'
+import h4 from '@public/webp/8-4.webp'
+import i1 from '@public/webp/9-1.webp'
+import i2 from '@public/webp/9-2.webp'
+import i3 from '@public/webp/9-3.webp'
+import i4 from '@public/webp/9-4.webp'
+import j1 from '@public/webp/10-1.webp'
+import j2 from '@public/webp/10-2.webp'
+import j3 from '@public/webp/10-3.webp'
+import j4 from '@public/webp/10-4.webp'
+
+import k1 from '@public/webp/11-1.webp'
+import k2 from '@public/webp/11-2.webp'
+import k3 from '@public/webp/11-3.webp'
+import k4 from '@public/webp/11-4.webp'
+import l1 from '@public/webp/12-1.webp'
+import l2 from '@public/webp/12-2.webp'
+import l3 from '@public/webp/12-3.webp'
+import l4 from '@public/webp/12-4.webp'
+import m1 from '@public/webp/13-1.webp'
+import m2 from '@public/webp/13-2.webp'
+import m3 from '@public/webp/13-3.webp'
+import m4 from '@public/webp/13-4.webp'
+import n1 from '@public/webp/14-1.webp'
+import n2 from '@public/webp/14-2.webp'
+import n3 from '@public/webp/14-3.webp'
+import n4 from '@public/webp/14-4.webp'
+import o1 from '@public/webp/15-1.webp'
+import o2 from '@public/webp/15-2.webp'
+import o3 from '@public/webp/15-3.webp'
+import o4 from '@public/webp/15-4.webp'
+import { ChevronLeft, ChevronRight } from 'lucide-react'
+
+// 첫번째 슬라이드
+// const webp1 = [a1, a2, a3, a4]
+// const webp2 = [b1, b2, b3, b4]
+// const webp3 = [c1, c2, c3, c4]
+// const webp4 = [d1, d2, d3]
+// const webp5 = [e1, e2, e3]
+
+// // 두번째 슬라이드
+// const webp6 = [f1, f2, f3, f4]
+// const webp7 = [g1, g2, g3, g4]
+// const webp8 = [h1, h2, h3, h4]
+// const webp9 = [i1, i2, i3, i4]
+// const webp10 = [j1, j2, j3, j4]
+
+// // 두번째 슬라이드
+// const webp11 = [k1, k2, k3, k4]
+// const webp12 = [l1, l2, l3, l4]
+// const webp13 = [m1, m2, m3, m4]
+// const webp14 = [n1, n2, n3, n4]
+// const webp15 = [o1, o2, o3, o4]
+
+const allWebps = [
+  a1,
+  a2,
+  a3,
+  a4,
+  b1,
+  b2,
+  b3,
+  b4,
+  c1,
+  c2,
+  c3,
+  c4,
+  d1,
+  d2,
+  d3,
+  e1,
+  e2,
+  e3,
+  f1,
+  f2,
+  f3,
+  f4,
+  g1,
+  g2,
+  g3,
+  g4,
+  h1,
+  h2,
+  h3,
+  h4,
+  i1,
+  i2,
+  i3,
+  i4,
+  j1,
+  j2,
+  j3,
+  j4,
+  k1,
+  k2,
+  k3,
+  k4,
+  l1,
+  l2,
+  l3,
+  l4,
+  m1,
+  m2,
+  m3,
+  m4,
+  n1,
+  n2,
+  n3,
+  n4,
+  o1,
+  o2,
+  o3,
+  o4,
+]
+
+console.log('allWebps.length()', allWebps.length)
+
+const OPTIONS: EmblaOptionsType = {
+  // slidesToScroll: 'auto',
+  // loop: true,
+  slidesToScroll: 5,
+  // active: false,
+  // containScroll: 'keepSnaps',
+  // containScroll: 'trimSnaps',
+}
+
+export default function Carousel() {
+  const [emblaRef, emblaApi] = useEmblaCarousel(OPTIONS)
+
+  const scrollPrev = React.useCallback(() => {
+    if (emblaApi) emblaApi.scrollPrev()
+    console.log('allWebps.length()', allWebps.length)
+  }, [emblaApi])
+
+  const scrollNext = React.useCallback(() => {
+    if (emblaApi) emblaApi.scrollNext()
+  }, [emblaApi])
+
+  return (
+    <section className="w-[1200px] mx-auto relative">
```

Comment:
현재 prev, next 버튼 클릭할 때마다 캐러셀의 위치가 다 달라요 (왼쪽 or 오른쪽으로 치우침)
`px-4` 적용하시면 해결됩니다~!
```suggestion
    <section className="px-4 w-[1200px] mx-auto relative">
```